src/main/java/bob/Task/Task.java

```diff
@@ -0,0 +1,35 @@
+package bob.Task;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.io.Serializable;
+
+/**
+ * Represents the task that the bob program will record.
+ */
+public abstract class Task implements Serializable {

```

Comment:
Pretty solid idea to declare Task as an abstract class and having the 3 task classes inherit from this👍

Reply:
Thank you! 😄